src/test/java/taub/citi/CitiBikeServiceTest.java

```diff
@@ -0,0 +1,155 @@
+package taub.citi;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+class CitiBikeServiceTest
+{
+    CitiBikeService service = new CitiBikeServiceFactory().getService();
+
+    @Test
+    void getStations()
+    {
+        Stations stations = service.getStations().blockingGet();
+        Station station1 = stations.data.stations.get(0);
+        assertNotNull(station1.station_type);
+        assertNotNull(station1.station_id);
+        assertNotNull(station1.name);
+    }
+
+    @Test
+    void getStatuses()
+    {
+
+        Stations stations = service.getStatuses().blockingGet();
+        Station station1 = stations.data.stations.get(0);
+        assertNotNull(station1.station_id);
+        assertNotNull(station1.num_docks_available);
+        assertNotNull(station1.last_reported);
+    }
+
+    @Test
+    void getStatusByStationId()
```

Comment:
You're not actually testing anything with this.

Try creating a method that does this and then having your test call the method.